### PR TITLE
[parquet-sdk][token_v2] migrate token_v2_pending_claims

### PR DESCRIPTION
**Pull request by yuunlimm**
### Description
- migrated pending claims

### TestPlan
ran both legacy and sdk postgres processor .. num of rows match for all three.

Screenshot 2024-12-11 at 8 00 13 PM
Screenshot 2024-12-11 at 9 25 27 PM
Screenshot 2024-12-11 at 9 26 37 PM

### CODE REVIEW

**Comment by yuunlimm** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/aptos-labs/aptos-indexer-processors/630?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#630** <a href="https://app.graphite.dev/github/pr/aptos-labs/aptos-indexer-processors/630?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈 <a href="https://app.graphite.dev/github/pr/aptos-labs/aptos-indexer-processors/630?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* **#626** <a href="https://app.graphite.dev/github/pr/aptos-labs/aptos-indexer-processors/626?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#625** <a href="https://app.graphite.dev/github/pr/aptos-labs/aptos-indexer-processors/625?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#619** <a href="https://app.graphite.dev/github/pr/aptos-labs/aptos-indexer-processors/619?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#616** <a href="https://app.graphite.dev/github/pr/aptos-labs/aptos-indexer-processors/616?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#615** <a href="https://app.graphite.dev/github/pr/aptos-labs/aptos-indexer-processors/615?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.

File: rust/sdk-processor/src/steps/parquet_token_v2_processor/parquet_token_v2_extractor.rs [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+            _current_token_ownerships_v2,
+            _current_deleted_token_ownerships_v2,
+            _token_activities_v2,
+            _current_token_v2_metadata,
+            _current_token_royalties_v1,
+            raw_current_token_claims,
+        ) = parse_v2_token(
+            &transactions.data,
+            &table_handle_to_owner,
+            &mut conn,
```
Comment (dermanyang) [inline rust/sdk-processor/src/steps/parquet_token_v2_processor/parquet_token_v2_extractor.rs:93]: I thought we were going to pass these as none?

File: rust/sdk-processor/src/parquet_processors/parquet_token_v2_processor.rs [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+        let transaction_stream = TransactionStreamStep::new(TransactionStreamConfig {
+            starting_version: Some(starting_version),
+            ..self.config.transaction_stream_config.clone()
+        })
+        .await?;
+
+        let backfill_table = set_backfill_table_flag(parquet_processor_config.backfill_table);
+        // TODO: Update this
+        let parquet_token_v2_extractor =
+            ParquetTokenV2Extractor::new(backfill_table, 5, 500, self.db_pool.clone());
```
Comment (dermanyang) [inline rust/sdk-processor/src/parquet_processors/parquet_token_v2_processor.rs:118]: How come these values aren't taken from the config?

File: rust/processor/src/db/common/models/token_v2_models/raw_token_claims.rs [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+    pub name: String,
+    pub amount: BigDecimal,
+    pub table_handle: String,
+    pub last_transaction_version: i64,
+    pub last_transaction_timestamp: chrono::NaiveDateTime,
+    pub token_data_id: String,
+    pub collection_id: String,
+}
+
+impl Ord for RawCurrentTokenPendingClaim {
```
Comment (dermanyang) [inline rust/processor/src/db/common/models/token_v2_models/raw_token_claims.rs:42]: I thought only the parquet models require `Ord` and `PartialOrd`?

File: rust/sdk-processor/src/parquet_processors/parquet_token_v2_processor.rs (hunk 2 above)
Comment (yuunlimm) [reply rust/sdk-processor/src/parquet_processors/parquet_token_v2_processor.rs:118]: those were just placeholders, and this code is updated in the stacked pr as well

File: rust/processor/src/db/common/models/token_v2_models/raw_token_claims.rs (hunk 3 above)
Comment (yuunlimm) [reply rust/processor/src/db/common/models/token_v2_models/raw_token_claims.rs:42]: sort is done for raw models in the parsing logic

**Review by dermanyang — APPROVED** [review]

**Pull request merged**